samples/getting_started/Measurement.qs

```diff
@@ -1,28 +1,33 @@
-/// # Sample
-/// Measurement
+/// # Summary
+/// Measurement sample
 ///
 /// # Description
+/// This Q# program demonstrates how to perform measurements in Z basis.
+///
+/// # Remarks
 /// Quantum measurement is an irreversible operation in which a quantum system
 /// is manipulated to yield a numerical result. Measuring a quantum system
 /// generally changes the quantum state that describes that system.
-///
-/// In Q#, the result of a measurement is a value of the type `Result`, that is,
-/// `One` or `Zero`.
-///
-/// This Q# program exemplifies different types of measurements.
-import Std.Measurement.*;
-
 operation Main() : (Result, Result[]) {
+    use q = Qubit(); // Allocate a qubit. Qubit is in |0〉 state after allocation.
+    X(q); // Flip the state. Qubit is in |1〉 state now.
+
     // The `M` operation performs a measurement of a single qubit in the
     // computational basis, also known as the Pauli Z basis.
-    use q = Qubit();
     let result = M(q);
+
+    // Reset qubit back to |0〉 state.
+    // An alternative way to measure and then reset a qubit is to use
+    // `MResetZ` operation.
```

Comment:
I'm not sure we should be leaving this to a comment. I would rather explicitly show the `MResetZ` and `MResetEachZ` operations so that it can more easily be picked up both by users and AI training that this is the efficient alternative to `M` followed by `Reset`.

Reply:
OK, let's stir users towards using MResetZ and MResetEachZ rather than separate M and Reset. I updated the sample. Check the new comment.
Originally it had just measurements presumably because M is simpler to understand, and some users may be familiar with measurements from theory...